model/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -379,7 +380,13 @@ String getLogHome() {
   }
 
   public void setLogHome(String logHome) {
-    this.logHome = logHome;
+    this.logHome = Optional.ofNullable(logHome).map(this::validatePath).orElse(null);
+  }
+
+  private String validatePath(String s) {
+    if (s.isBlank()) return null;
+    if (s.endsWith(File.separator)) return s;
+    return s + File.separator;
```

Comment:
Should we require that the path start with a separator, or just add one if its not there?

Reply:
I suppose we could apply a general "is this a valid path" logic to all such fields. That would include the leading separator and probably more.

Reply:
After experimenting with some validation, I'm not sure that we can count on knowing what the start of a valid path has to be; it will depend on the mapped resource. Windows using a different initial string than does Linux. The only thing we can be sure of is that the trailing separator is significant.

Reply:
Does this help?
```File file = new File(path);
if (file.isAbsolute()) {```

Reply:
Yes, that works. New test and validation added.